databases/week2/queries.sql

```diff
@@ -0,0 +1,173 @@
+--Part 1: Working with tasks
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO
+  task(
+    title,
+    description,
+    created,
+    updated,
+    due_date,
+    status_id,
+    user_id
+  )
+VALUES(
+    'Homework',
+    'Complete this week homework',
+    NOW(),
+    NOW(),
+    '2021-08-05''10:01:25',
+    1,
+    1
+  );
+--Change the title of a task
+UPDATE
+  task
+SET
+  title = 'Complete Homework'
+WHERE
+  id = 36;
+--Change a task due date
+UPDATE
+  task
+SET
+  due_date = '2021-08-06''13:01:25'
+WHERE
+  id = 36;
+--Change a task status
+UPDATE
+  task
+SET
+  status_id = 2
+WHERE
+  id = 36;
+--Mark a task as complete
+UPDATE
+  task
+SET
+  status_id = 3
+WHERE
+  id = 36;
+--Delete a task
+DELETE FROM
+  task
+WHERE
+  id = 36;
+--Part 2: School database
+  --Create a new database containing the following tables:
+  --Class: with the columns: id, name, begins (date), ends (date)
+  --Student: with the columns: id, name, email, phone, class_id (foreign key)
+  CREATE database school_db;
+USE school_db;
+CREATE Table `class`(
```

Comment:
It is nice to have the creation of your database structures in a separate file, it makes it easier to reason about it

Reply:
Ok- I will do that in seperate file.